.travis.yml

```diff
@@ -1,22 +1,16 @@
-sudo: false
 language: java
 jdk:
-- openjdk8
+- openjdk11
 install: "./gradlew assemble"
 script: gradle externalCiTest
 before_cache:
 - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
 - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
 cache:
-  apt: true
   directories:
   - "$HOME/.m2"
   - "$HOME/.gradle/caches/"
   - "$HOME/.gradle/wrapper/"
 before_install:
 - openssl aes-256-cbc -K $encrypted_4f1b6fd74eac_key -iv $encrypted_4f1b6fd74eac_iv
   -in gradle.properties.enc -out gradle.properties -d
-after_success:
-  - test $TRAVIS_BRANCH = "master" && ./gradlew uploadArchives
```

Comment:
is there a reason for not wanting travis to make the publishes?

Reply:
Just, this is an application, not a library. We are not publishing our apps to sonatype.